src/selection.js

```diff
@@ -164,6 +164,17 @@ export const findPositionOfNodeBefore = selection => {
 // ```
 export const findDomRefAtPos = (position, domAtPos) => {
   const dom = domAtPos(position);
-  const node = dom.node.childNodes[dom.offset];
+  let { offset } = dom;
+  for (let i = 0, count = dom.node.childNodes.length; i < count; i++) {
```

Comment:
If I have something like:
```
let { offset, node } = dom;
offset = 1;
node.childNodes = [decoration, decoration, myThing, decoration, decoration];
```
won't this set the offset to `5`?

Reply:
I'll add a test for it

Reply:
all done